doc/classes/CanvasItem.xml

```diff
@@ -576,7 +576,7 @@
 			Disable blending mode. Colors including alpha are written as is. Only applicable for render targets with a transparent background. No lighting will be applied.
 		</constant>
 		<constant name="NOTIFICATION_TRANSFORM_CHANGED" value="29">
-			Canvas item transform has changed. Only received if requested.
+			Canvas item transform has changed. Notification is only received if enabled by set_notify_transform() or set_notify_local_transform().
```

Comment:
Please use method references so links will be generated, ie  "[method set_notify_transform]".

Reply:
Thanks, I'll amend that.  X)